client/lib/reader-lists-feeds/store.js

```diff
@@ -0,0 +1,91 @@
+// Reader Lists Feed Subscription Store
+
+// External dependencies
+import { List, Map, fromJS } from 'immutable';
+import debugModule from 'debug';
+
+// Internal dependencies
+import { action as actionTypes } from './constants';
+import { createReducerStore } from 'lib/store';
+
+const debug = debugModule( 'calypso:reader-lists-feeds' ); //eslint-disable-line no-unused-vars
+
+const initialState = {
+	feeds: Map(), // eslint-disable-line new-cap
```

Comment:
why `Map` here and `{}` everywhere else? they all end up getting converted to `Map` with the `fromJS` call later.